### PR TITLE
NRPT-202: Cleanup ROLES constants

### PR DESCRIPTION
**Pull request by marklise**
Everything sits in the api folder now, possible next improvement is to push the valid roles config into the back-end and query on SPA initialize.

### CODE REVIEW

File: api/src/utils/task-audit-record.test.js [hunk 1]
```diff
@@ -52,11 +52,11 @@ describe('TaskAuditRecord', () => {
 
       expect(mockFindOneAndUpdate).toHaveBeenCalledWith(
         { _id: '456' },
-        { $set: { _id: '456', ...{ param1: 3, param2: 4, read: ROLES.ADMIN_ROLES, write: [ROLES.SYSADMIN] } } },
+        { $set: { _id: '456', ...{ param1: 3, param2: 4, read: utils.ApplicationRoles.ADMIN_ROLES, write: [utils.ApplicationRoles.SYSADMIN] } } },
```
Comment (KitArmstrong) [inline api/src/utils/task-audit-record.test.js:55]: Some of these are still using ADMIN_ROLES which was removed.

File: api/src/utils/constants/misc.js [hunk 2]
```diff
@@ -1,19 +1,11 @@
...
-    'sysadmin',
-    'admin:lng',
-    'admin:nrced',
-    'admin:bcmi'
-  ]
-};
+exports.ApplicationRoles = {
+  ADMIN: 'sysadmin',
+  ADMIN_NRCED: 'admin:nrced',
+  ADMIN_LNG: 'admin:lng',
```
Comment (KitArmstrong) [inline api/src/utils/constants/misc.js:6]: Need to add ADMIN_ROLES back or change everywhere it is used.

Comment (marklise) [reply api/src/utils/constants/misc.js:6]: I'll change how it works.

Comment (marklise) [reply api/src/utils/constants/misc.js:6]: Fixed in 59a3d4f

File: api/src/utils/task-audit-record.test.js (hunk 1 above)
Comment (marklise) [reply api/src/utils/task-audit-record.test.js]: Fixed in 59a3d4f.

**Pull request merged**